OJP/OJP_FareSupport.xsd

```diff
@@ -8,6 +8,20 @@
 	<xs:annotation>
 		<xs:documentation>========================================== Fare Authority ==========================================</xs:documentation>
 	</xs:annotation>
+	<xs:complexType name="BookingNoteStructure">
+		<xs:annotation>
+			<xs:documentation>A structure for booking notes.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="BookingNote" type="BookingNoteType" minOccurs="1" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="BookingNoteType">
+		<xs:annotation>
+			<xs:documentation>An element of the BookingNotes or as a single BookingNote.</xs:documentation>
+		</xs:annotation>
+		<xs:attribute name="order" type="xs:positiveInteger"/>
```

Comment:
Please do not add `xs:attribute`s.
Is a `order` really necessary and thus having the client re-order the notes? The response should contain the notes in correct order - for trips and other places we are able to do so as well..

Reply:
@skinkie no order for you ok then?

Reply:
@sgrossberndt lets discuss this afternoon. I don't like the implicit ordering idea.

Reply:
exists in InternationalTextStructure already.

Reply:
order was removed